Sources/Networking/Responses/GetIntroEligibilityResponse.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  GetIntroEligibilityResponse.swift
+//
+//  Created by Nacho Soto on 5/12/22.
+
+import Foundation
+
+struct GetIntroEligibilityResponse {
+
+    var eligibilityByProductIdentifier: [String: IntroEligibilityStatus]
+
+}
+
+extension GetIntroEligibilityResponse: HTTPResponseBody {
```

Comment:
Because we use a dedicated protocol `HTTPResponseBody` we can implement a custom deserialization in this case.